Runtime/Scripts/MobiledgeXWebSocketClient.cs

```diff
@@ -108,9 +89,14 @@ public void Send(string message)
             byte[] buffer = encoder.GetBytes(message);
             //Debug.Log("Message to queue for send: " + buffer.Length + ", message: " + message);
             var sendBuf = new ArraySegment<byte>(buffer);
-
             sendQueue.Add(sendBuf);
         }
+       public async void Send(byte[] binary)
+        {
+            //Debug.Log("Message to queue for send: " + buffer.Length + ", message: " + message);
+            var sendBuf = new ArraySegment<byte>(binary);
+            await ws.SendAsync(sendBuf, WebSocketMessageType.Binary, true /* is last part of message */, CancellationToken.None);
```

Comment:
Is there no sendQueue for binary data?

Reply:
I wanted to use the MobiledgeXWebSocketClient in the CV app so small change to send, the sending queue is of type string.

Reply:
The websocket client has independent queues for send and receive, await would freeze the UI sending that binary buffer.

Reply:
@lgarner  @franklin-huang-mobiledgex  I added another queue for binary messages and an independent thread (sendThreadBinary) that is being initialized at the constructor.